### ISSUE TITLE
Mock up multiple determination interviews scenario

### ISSUE THREAD

**Issue by nicoleslaw** [cagov/ui-claim-tracker#246]
### Description

Depends on #245 

### Acceptance Criteria

- [ ] Mock up what the claim tracker looks like when the claimant has multiple determination interviews
- [ ] Share with Diana and Nicole for review

**Comment by nicoleslaw** [comment]
Closing since this has been descoped for MVP.
